perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/GeneralServiceManagerMethod.java

```diff
@@ -21,7 +21,8 @@
 	 */
 	blockServiceOnFacility {
 	    public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
-		ac.getGeneralServiceManager().blockServiceOnFacility(ac.getSession(), ac.getServiceById(parms.readInt("service")),
+
```

Comment:
Please add ``@deprecated`` line in javadoc of all methods of this manager in RPC.

Reply:
Please also specify info about deprecation like ``@deprecated Method was moved to ServicesManager``. I will extend our RPC docs parsing tool to include this message.